docs/site/Relation-generator.md

```diff
@@ -0,0 +1,106 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+# Relation generator
+
+### Prerequisites
+
+Important: Before running this generator, you must create an application using
+the application generator. Then you must run the command from the root directory
+of the application. The models involved in the relation must exist before
+running this generator. The datasource involved in the relation must exist
+before running this generator. The repositories involved in the relation must
+exist before running this generator.
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+## Creates a new LoopBack4 relation.
+
+Adds a new `Relation` between existing source and target models in a LoopBack
+application.
+
+```sh
+lb4 relation [options]
+```
+
+### Options
+
+- `--help`: Print the generator’s options and usage.
+- `--skip-cache`: Do not remember prompt answers. Default: `false`.
+- `--skip-install`: Do not automatically install dependencies. Default: `false`.
+- `--force-install`: Fail on install dependencies error. Default: `false`.
+- `--relationType`: Relation type.
+- `--sourceModel`: Source model.
+- `--destinationModel`: Destination model.
+- `--foreignKeyName`: Destination model foreign key name.
+- `--relationName`: Relation name.
+
+### Arguments
+
+Defining lb4 relation in one command line interface (cli):
+
+\$ lb4 relation --sourceModel=`<sourceModel>`
+--destinationModel=`<destinationModel>` --foreignKeyName=`<foreignKeyName>`
+--relationType=<hasMany|belongsTo>[--relationName=`<relationName>`]
+
+`<relationType>` - Type of the relation that will be created between the source
+and target models.
+
+`<sourceModel>` - Name of the model to create the relationship from.
```

Comment:
```suggestion
- `<sourceModel>` - Name of the model to create the relationship from.
```

The same comment applies to other lines below too.

Reply:
Fixed.